### ISSUE TITLE
Pre-commit fails to find pip

### ISSUE THREAD

**Issue by apatard** [pre-commit/pre-commit#1433]
I ran precommit today in zuul-ci and pre-commit failed to find pip, so the CI check failed.:
```
An unexpected error has occurred: CalledProcessError: command: ('/home/zuul/.cache/pre-commit/repofffvfvpt/py_env-python3/bin/python', '-mpip', 'install', '.')
return code: 1
expected return code: 0
stdout: (none)
stderr:
    /home/zuul/.cache/pre-commit/repofffvfvpt/py_env-python3/bin/python: No module named pip
```

The logs and jobs information are available at https://dashboard.zuul.ansible.com/t/ansible/build/c098bbf584fa4831881cd6be80daa56a.

**Comment by ssbarnea** [comment]
I wonder if we can get more useful information about the failure if you call it using `pre-commit run -v -a` (more verbose).

**Comment by apatard** [comment]
I can add it but looks like it's working now, with no change in the CI in the project (same PR, same source code). Not sure what to do now.

**Comment by asottile** [comment]
I suspect this was related to the virtualenv 20.0.19 release which was quickly fixed by 20.0.20

since it's no longer reproducing I'm going to close this, but if it comes up again please comment here -- thanks!
